gems/pending/postgres_ha_admin/postgres_ha_logger.rb

```diff
@@ -0,0 +1,23 @@
+module PostgresHaAdmin
+  module PostgresHaLogger
+    attr_reader :log_file
+
+    def init_logger(log_dir)
+      @log_file = Pathname.new(log_dir).join('ha_admin.log')
+    end
+
+    def log_info(message)
+      File.open(log_file, 'a') do |f|
+        f.write "[----] I, [#{Time.now.utc.iso8601(6)} ##{Process.pid}:"\
+            "#{Thread.current.object_id.to_s(16)}]  INFO -- : #{self.class.name} - #{message}"
+      end
```

Comment:
yes, it is not good. I was planning to refactor it later after main functionality merged (since it is outside of main purpose of this PR)